modules/gcp-gce/init.tpl

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# install Docker
+curl -fsSL https://get.docker.com -o get-docker.sh
+sh get-docker.sh
+
+# install Docker Compose
+curl -L "https://github.com/docker/compose/releases/download/v2.27.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+chmod +x /usr/local/bin/docker-compose
+
+cd /home/${user}
+
+# clone repo
+git clone https://github.com/onebusaway/onebusaway-docker.git
+
+cd /home/${user}/onebusaway-docker
+
+# create .env file
+echo "${docker_env}" > .env
+
+
+# onebusaway-api-webapp depends on mysql, normally this will handle by docker-compose
+# but in Azure, the mysql container will not be ready when onebusaway-api-webapp starts
```

Comment:
Your comment refers to Azure, but this PR is about GCP. Should the comment be updated, or should this DB initialization stanza be removed?

Reply:
I will update the comment, MySql container sometimes acts quirky, therefore starting it first will prevent these problems.